src/azure-cli-core/azure/cli/core/commands/client_factory.py

```diff
@@ -161,7 +161,10 @@ def _get_mgmt_service_client(cli_ctx,
     from azure.cli.core._profile import Profile
     logger.debug('Getting management service client client_type=%s', client_type.__name__)
     resource = resource or cli_ctx.cloud.endpoints.active_directory_resource_id
-    profile = Profile(cli_ctx=cli_ctx)
+
+    client_id = kwargs.pop('client_id', None)
+
+    profile = Profile(cli_ctx=cli_ctx, scopes=scopes, client_id=client_id)
```

Comment:
I will merge it first but this may be changed because `Profile` does 2 things: `login` and `get_login_credentials` which could use different `scopes`.